src/outpace/remote/service.cljc

```diff
@@ -0,0 +1,308 @@
+(ns outpace.remote.service
+  #?(:clj (:require [clojure.walk :as walk]
+                    [clojure.core.async :as async :refer [go]]
+                    [clj-http.client :as http]
+                    [clojure.string :as string]))
+  #?@(:cljs
+      [(:require [cljs-http.client :as http]
+                 [cljs.core.async :as async]
+                 [clojure.string :as string])
+       (:require-macros
+        [cljs.core.async.macros :refer [go]])]))
+
+
+;; ---------------------------------------------------------------------
+;; Utilities
+
+#?(:clj
```

Comment:
Would it be better to push this reader conditional further down into the defn since most of the body is the same?